core/sodasql/cli/cli.py

```diff
@@ -189,7 +189,13 @@ def matches_table_exclude(table_name: str, table_exclude_pattern):
 @click.option('-e', '--exclude',
               required=False,
               help='Table name exclusion filter, case insensitive, comma separated list, use * as a wild card')
-def analyze(warehouse_file: str, include: str, exclude: str):
+@click.option('-l', '--limit',
+              default=10,
```

Comment:
Why do we want a global default limit of 10? This sounds like a breaking change that will produce unexpected results for users using everything else than snowflake

Reply:
Good point.

This is not done yet - I'm adding limit option to all the dialects, and remove the default 10 on snowflake as well.